contrib/adduser.c

```diff
@@ -398,7 +398,7 @@ main (void)
 
   *environ = NULL;
 
-  bzero (cmd, sizeof (cmd));
+  bzero(cmd, sizeof(cmd));
```

Comment:
@thesamesam 

If you apply the semantic patch accompanied with the corresponding commit message, and then `git diff -w` against the actual commit, you should be able to ignore these unrelated whitespace changes.  I've written recommendations for how to better review each commit.

I hope that helps reviewing, while at the same time allowing me to apply some house style while doing the changes.